CHANGELOG.txt

```diff
@@ -5,6 +5,8 @@ Changes:
  - obspy.core:
    * add option to suppress evalresp sensitivity mismatch warning when removing
      instrument response (see #2677)
+ - obspy.clients.fdsn:
+   * introduce fine-grained FDSN client exceptions (see #2653)
    * round magnitudes in Catalog/Event string representation to one decimal
```

Comment:
This ended up in the middle of the `obspy.core` block, somehow. Please move it down.